src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Dropdown as ReactBootstrapDropdown } from "react-bootstrap";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+
+import { ADMIN_V1_ROOMS_URL } from "settings";
+
+import { Room } from "types/rooms";
+import { ContainerClassName } from "types/utility";
+
+import { venueSpacesList } from "utils/room";
+
+import { Dropdown } from "components/atoms/Dropdown";
+
+import "./SpacesDropdown.scss";
+
+export interface SpacesDropdownProps extends ContainerClassName {
```

Comment:
```suggestion
export interface SpacesDropdownProps {
```

Reply:
fixed in dc3dcc064168e6d2acbd386f35b99fa9f36961a1